.github/workflows/CI.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Install dependencies
       run: |
-        apt-get update && apt-get install -y libhdf5-dev libzmq3-dev python3-pip
+        apt-get update && apt-get install -y libhdf5-dev libzmq3-dev python3-pip hdf5-tools
```

Comment:
nit, but I might put `hdf5-tools` next to `libhdf5-dev` since they're logically related

Reply:
agree, will make change